taxi/models.py

```diff
@@ -15,7 +20,15 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=255,
+        unique=True,
+        validators=[
+            MinLengthValidator(8),
+            MaxLengthValidator(8),
+            RegexValidator(r"^[A-Z]{3}\d{5}$"),
```

Comment:
do you need to have here min and max validators if you regex has ^ and & with 8 characters between them?